dev/tools/controllerbuilder/pkg/options/generateoptions.go

```diff
@@ -32,7 +34,10 @@ func (o *GenerateOptions) InitDefaults() error {
 	if err != nil {
 		return nil
 	}
-	o.ProtoSourcePath = root + "/.build/googleapis.pb"
+	o.ProtoSourcePath, err = ProtoSourcePath(root)
```

Comment:
nit: rename to `defaultProtoSourcePath`. We might also want to move this to a later stage where the user input is bound to the flag. Perhaps we could generate it when we attempt to read it and find that it doesn't exist.

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/7bb291b13175d8614a98d6acc539474fb332344f/dev/tools/controllerbuilder/pkg/protoapi/loader.go#L33